docs/pages/developer-guide.md

```diff
@@ -1221,6 +1221,23 @@ pods:
       ....
 ```
 
+# Regions and Zones
+
+Mesos allows agents to expose fault domain information in the form of a region and zone. A region is larger than a zone and should be thought of as containing zones. For example, a region could be a particular datacenter and the racks within that datacenter could be its zones. When this information is provided by Mesos, it is injected into each task's environment. For example:
+
+```
+REGION: us-west-2
+ZONE: us-west-2a
+```
+
+Services may choose to use this information to enable rack awareness. When doing so, they should use placement rules to ensure that their pods are appropriately placed withing regions and zones. One may apply placement constraints against regions and zones by referencing `@region` and `@zone` keys.  For example:
+
+```
+@zone:GROUP_BY:2
+```
+
+The placement rule above would apply the `GROUP_BY` operator to zones.  The SDK does not currently enable region awareness, so any given service will only receive offers in the local region.
```

Comment:
@triclambert and @crackcode123: Note the addition of the last sentence.  Otherwise I just moved the whole section to fix this section accidentally being between the TLS section and its children.